src/Holding/ViewHolding/HoldingBoundWith/fixtures.js

```diff
@@ -0,0 +1,7 @@
+// Disabling eslint below because the pattern is to name fixtures
```

Comment:
I think this is correct, but please correct me if not.

Reply:
Also, files named like `...test.js` will be excluded from the production build. Fixture data should be similarly excluded but we probably need to add a new rule for that, or move them into an already excluded directory such as `/test/` or `.../test/...`.